source/guide_guillotine.rst

```diff
@@ -0,0 +1,280 @@
+.. highlight:: console
+
+.. author:: Jan Philip Bernius <https://janphilip.bernius.net>
+
+.. tag:: lang-ruby
+
+##########
+Guillotine
+##########
+
+.. tag_list::
+
+Guillotine_ is a simple URL Shortener written in Ruby and powers GitHubs URL Shortener Git.io_.
+It supports multiple storage adapters, including :manual:`MySQL <database-mysql>`, :lab:`PostgreSQL <guide_postgresql>`, SQLite, :lab:`Redis <guide_redis>`, Riak and Cassandra.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`Ruby <lang-ruby>` and its package manager :manual_anchor:`gem <lang-ruby.html#gem>`
+  * :manual:`MySQL <database-mysql>`
+  * :manual:`supervisord <daemons-supervisord>`
+  * :manual:`domains <web-domains>`
+
+License
+=======
+
+Guillotine is released under the `MIT License`_.
+
+Prerequisites
+=============
+
+We're using :manual:`Ruby <lang-ruby>` in the stable version 10:
+
+.. code-block:: console
+
+  [isabell@stardust ~]$ uberspace tools version use ruby 2.6
+  Selected Ruby version 2.6
+  The new configuration is adapted immediately. Patch updates will be applied automatically.
+
+  [isabell@stardust ~]$
+
+.. include:: includes/my-print-defaults.rst
+
+Your Short URL needs to be setup:
+
+.. include:: includes/web-domain-list.rst
+
+Installation
+============
+
+Install Dependencies
+--------------------
+
+Use ``gem`` to install the latest versions of the Guillotine, Sequel and mysql2 gems:
+
+.. code-block:: console
+  :emphasize-lines: 1
+
+  [isabell@stardust ~]$ gem install guillotine sequel mysql2
+  Fetching sinatra-1.4.8.gem
+  Fetching addressable-2.3.8.gem
+  Fetching rack-1.6.11.gem
+  Fetching tilt-2.0.9.gem
+  Fetching rack-protection-1.5.5.gem
+  Fetching guillotine-1.4.2.gem
+  WARNING:  You don't have /home/isabell/.gem/ruby/2.6.0/bin in your PATH,
+  	  gem executables will not run.
+  Successfully installed rack-1.6.11
+  Successfully installed rack-protection-1.5.5
+  Successfully installed tilt-2.0.9
+  Successfully installed sinatra-1.4.8
+  Successfully installed addressable-2.3.8
+  Successfully installed guillotine-1.4.2
+  Fetching sequel-5.22.0.gem
+  Successfully installed sequel-5.22.0
+  Fetching mysql2-0.5.2.gem
+  Building native extensions. This could take a while...
+  Successfully installed mysql2-0.5.2
+  8 gems installed
+
+  [isabell@stardust ~]$
+
+Create App File
+---------------
+
+Create a directory, and inside the directory a file ``app.rb`` for the application:
+
+.. code-block:: console
+  :emphasize-lines: 1-2
+
+  [isabell@stardust ~]$ mkdir ~/guillotine
+  [isabell@stardust ~]$ touch ~/guillotine/app.rb
```

Comment:
In addition to `app.rb`, a `config.ru` is also created a few steps down. Please either touch all required files beforehand, or none (my favorite).

Reply:
Removed this section.